.github/workflow/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```

Comment:
- [ ] Please make sure that setup linters correctly 
![image](https://user-images.githubusercontent.com/74744787/184976733-b8a8d9b4-b59a-4ec1-a2a8-0c0c97844d40.png)